flake.nix

```diff
@@ -2,29 +2,30 @@
   inputs.nixpkgs.url = "github:nixos/nixpkgs";
   inputs.flake-utils.url = "github:numtide/flake-utils";
 
-  outputs = { self, nixpkgs, flake-utils, ... }@inputs:
+  outputs = { nixpkgs, flake-utils, ... }:
     flake-utils.lib.eachDefaultSystem (system:
       let
         pkgs = import nixpkgs { inherit system; };
-        shellPackages = [
-          "jre"
-          "sbt"
-          "nodejs-16_x"
-          "yarn"
+        shellPackages = with pkgs; [
+          temurin-jre-bin-17
```

Comment:
**thought:** I replaced default `jre` with the same one that is used on the [CI](https://github.com/disneystreaming/smithy4s/blob/series/0.18/.github/workflows/ci.yml#L63)